### DISCUSSION TITLE
Checkbox in theme

### DISCUSSION THREAD

**Discussion by NB071** [mantinedev/mantine#5497] [Feature requests]
Hey. I was trying to set a default color for checkbox in the theme object ;however it wasn't possible. Unfortunately, Checkbox component doesn't have `.extend` method . Could you please add it or recommend some alternative in the time of your implementation. Thanks!

**Comment by rtivital** [comment]
It works fine on codesandbox – https://codesandbox.io/s/musing-chihiro-pj66tj?file=/src/App.tsx
Make sure that you are using the latest version
